src/solvers/dgsem_structured/dg_3d.jl

```diff
@@ -98,8 +98,9 @@ See also https://github.com/trixi-framework/Trixi.jl/issues/1671#issuecomment-17
 
         # Compute the contravariant flux by taking the scalar product of the
         # third contravariant vector Ja^3 and the flux vector
-        Ja31, Ja32, Ja33 = get_contravariant_vector(3, contravariant_vectors, i, j, k,
-                                                    element)
+        Ja31, Ja32,
+        Ja33 = get_contravariant_vector(3, contravariant_vectors,
```

Comment:
The newline behind `Ja33` is most likely not by the formatter

Reply:
I think JuliaFormatter v2 does such things. When applying the autoformatter locally, remember to use the same version as our CI (v1.0.60), cf. https://trixi-framework.github.io/Trixi.jl/stable/styleguide/#automated-source-code-formatting.

Reply:
Oh man